src/fedora/36/amd64/Dockerfile

```diff
@@ -51,3 +51,9 @@ RUN dnf --setopt=install_weak_deps=False install -y \
         uuid-devel \
         zlib-devel \
     && dnf clean all
+
+# Dependencies for running the VMR tooling
+# LibGit2Sharp requires this when synchronizing the VMR in a Codespace
+RUN dnf --setopt=install_weak_deps=False install -y \
+        openssl1.1.x86_64 \
```

Comment:
The `.x86_64` portion of this is implied based on the architecture of the container. I think it'd be better to omit this just for readability.

Reply:
Thanks!

```suggestion
        openssl1.1 \
```